hasher-matcher-actioner/Makefile

```diff
@@ -38,5 +38,11 @@ dev_create_instance: dev_create_configs upload_docker
 dev_destroy_instance:
 	cd terraform/ && terraform destroy -var "prefix=${USER}"
 
+dev_test_instance:
+	@echo "Attempting to run submit_content_test, due to make limitations you may need to run additional commands first."
+	@echo "This simple tests should take a little over 2 minutes to complete (due to sqs timeout)." 
+	@echo
```

Comment:
Nit: Recommend merging this into the output of the command itself.

If only valid during certain runs, trigger it with a --flag, and call that flag here.